migrations/v4.sql

```diff
@@ -0,0 +1,22 @@
+PRAGMA foreign_keys = OFF;
+
+-- Remove FOREIGN KEY constraint from ip_bans
+CREATE TABLE ip_bans_new(
+	ipid TEXT PRIMARY KEY,
+	ban_id INTEGER NOT NULL,
+	FOREIGN KEY (ban_id) REFERENCES bans(ban_id)
+		ON DELETE CASCADE
+);
+INSERT INTO ip_bans_new SELECT * FROM ip_bans;
+DROP TABLE ip_bans;
+ALTER TABLE ip_bans_new RENAME TO ip_bans;
+
+-- Add unbanned column
+ALTER TABLE bans ADD unbanned INTEGER DEFAULT 0;
```

Comment:
Hmm. Wondering if it makes more sense to invert this column, e.g. `active`, or to use the unban date directly to figure out if the ban is still in effect.

Reply:
https://github.com/AttorneyOnline/tsuserver3/pull/98#issuecomment-745641340